src/index.html

```diff
@@ -29,7 +29,30 @@ <h1>What's Cookin?</h1>
       <section class="center-column side-tag-display">
         <h1>Filter by Tag</h1>
         <article class="tag-list">
-        </article>
+        <input type="checkbox" name="all-tags" value="antipasti" id="antipastiTag">
+            <label for="antipastiTag">#antipasti</label><br><input type="checkbox" name="all-tags" value="antipasto" id="antipastoTag">
+            <label for="antipastoTag">#antipasto</label><br><input type="checkbox" name="all-tags" value="appetizer" id="appetizerTag">
+            <label for="appetizerTag">#appetizer</label><br><input type="checkbox" name="all-tags" value="breakfast" id="breakfastTag">
+            <label for="breakfastTag">#breakfast</label><br><input type="checkbox" name="all-tags" value="brunch" id="brunchTag">
+            <label for="brunchTag">#brunch</label><br><input type="checkbox" name="all-tags" value="condiment" id="condimentTag">
+            <label for="condimentTag">#condiment</label><br><input type="checkbox" name="all-tags" value="dinner" id="dinnerTag">
+            <label for="dinnerTag">#dinner</label><br><input type="checkbox" name="all-tags" value="dip" id="dipTag">
+            <label for="dipTag">#dip</label><br><input type="checkbox" name="all-tags" value="hor d'oeuvre" id="hor d'oeuvreTag">
+            <label for="hor d'oeuvreTag">#hor d'oeuvre</label><br><input type="checkbox" name="all-tags" value="lunch" id="lunchTag">
+            <label for="lunchTag">#lunch</label><br><input type="checkbox" name="all-tags" value="main course" id="main courseTag">
+            <label for="main courseTag">#main course</label><br><input type="checkbox" name="all-tags" value="main dish" id="main dishTag">
+            <label for="main dishTag">#main dish</label><br><input type="checkbox" name="all-tags" value="morning meal" id="morning mealTag">
+            <label for="morning mealTag">#morning meal</label><br><input type="checkbox" name="all-tags" value="salad" id="saladTag">
+            <label for="saladTag">#salad</label><br><input type="checkbox" name="all-tags" value="sauce" id="sauceTag">
+            <label for="sauceTag">#sauce</label><br><input type="checkbox" name="all-tags" value="side dish" id="side dishTag">
+            <label for="side dishTag">#side dish</label><br><input type="checkbox" name="all-tags" value="snack" id="snackTag">
+            <label for="snackTag">#snack</label><br><input type="checkbox" name="all-tags" value="spread" id="spreadTag">
+            <label for="spreadTag">#spread</label><br><input type="checkbox" name="all-tags" value="starter" id="starterTag">
+            <label for="starterTag">#starter</label><br>
+          </article>
```

Comment:
Something to keep in mind for the rest of the page, iterating over a querySelectorAll requires all nodes to exist